lib/server/controllers/qcFlag.controller.js

```diff
@@ -248,6 +248,36 @@ const getQcFlagsSummaryHandler = async (req, res) => {
     }
 };
 
+// eslint-disable-next-line valid-jsdoc
+/**
+ *
+ */
+const setGaqDetectors = async (req, res) => {
+    const validatedDTO = await dtoValidator(
+        DtoFactory.bodyOnly(Joi.object({
+            dataPassId: Joi.number(),
+            runNumbers: Joi.array().items(Joi.number()).unique(),
```

Comment:
From what you described, you will update GAQ only for runs one by one, no need to take an array here.

Reply:
From what I described for first feature in fronted , it is expected to do in batches from browser in the future and directly via API now as users use their own scripts, etc., so there is not place for such a simplification here